accounting/overview/process_overview/vendor_bill.rst

```diff
@@ -0,0 +1,139 @@
+==================================
+From Vendor Bills to Payments
+==================================
+
+Once vendor bills are registered in Odoo, you can easily pay vendors for
+the correct amount and at the right time (not too late, not too early;
+depending on your vendor policy). Odoo also offers reports to track your
+aged payable balances.
+
+If you want to control vendor bills received from your vendors, you can
+use the Odoo Purchase application that allows you to control and
+pre-complete them automatically based on past purchase orders.
+
+From Vendor Bill to Payment
+===========================
+
+Record a new vendor bill
+------------------------
+
+When a vendor bill is received, you can record it from the Vendor Bills
+submenu under Purchases in the Accounting application. As a shortcut,
+you can also use the “New Bill” feature on the accounting dashboard.
+
+.. image:: ./media/vendor_bill01.png
+   :align: center
+
+To register a new vendor bill, start by selecting a vendor and inputting
+their invoice as the Vendor Reference, then add and confirm the product
+lines, making sure to have the right product quantities, taxes and
+prices.
+
+.. image:: ./media/vendor_bill02.png
+   :align: center
+
+Save the invoice to update the pre tax and tax amounts at the bottom of
+the screen.You will most likely need to configure the prices of your
+products without taxes as Odoo will compute the tax for you.
+
+.. note:: 
+    On the bottom left corner, Odoo shows a summary table of all taxes on the vendor bill. In several countries, different methods are accepted to round the totals (round per line, or round globally). The default rounding method in Odoo is to round the final prices per line (as you may have different taxes per product. E.g. Alcohol and cigarettes). However if your vendor has a different tax amount on their bill, you can change the amount in the bottom left table to adjust and match.
+
+Validate The Vendor Bill
+------------------------
+
+Once the vendor bill is validated, a journal entry will be generated
+based on the configuration on the invoice. This journal entry may differ
+depending on the the accounting package you choose to use.
+
+For most European countries, the journal entry will use the following
+accounts:
+
+-  **Accounts Payable: *defined on the vendor form***
```

Comment:
just ` **Accounts Payable**: defined on the vendor form` otherwise looks like this
![screenshot from 2015-10-27 09-16-24](https://cloud.githubusercontent.com/assets/564822/10752647/68746bdc-7c8b-11e5-82f4-c4e9d198cfc4.png)